uiowa/uiowa
# SiteNow Dispatch Module

Resolves #3302 

## To test

Sync down a site
Add the block
enter a non-uiowa email and first & last name
check the ITS email that something got sent through
[Verify here](https://apps.its.uiowa.edu/dispatch/populations/718296392/members) that the email got added to the list.
Delete the example email that got added so it does not clutter the list
?????
Profit.

### \*\*hint\*\*
Ask Alan for an API key
